### ISSUE TITLE
 Update ESU form with button type submit 

### ISSUE THREAD

**Issue by LouiaR** [comicrelief/component-library#266]
Issue:  This form does not contain a submit button, which creates issues for those who cannot submit the form using the keyboard. Submit buttons are INPUT elements with type attribute "submit" or "image", or BUTTON elements with type "submit" or omitted/invalid.
https://headless-staging.comicrelief.com/rednoseday/schools/nurseries/
